src/parser.y

```diff
@@ -316,8 +316,8 @@ const char* utap_msg(const char *msg)
 %left T_MULT T_DIV T_MOD
 %left T_POWOP
 %right T_EXCLAM T_KW_NOT UOPERATOR
-%left '(' ')' '[' ']' '.' '\''
 %right T_INCREMENT T_DECREMENT
```

Comment:
The uniform treatment of `++` and `--` is not good (could it be the root of the issue?):
post-increment/decrement has left-to-right associativity
pre-increment/decrement has right-to-left associativity

We probably need to adjust expression grammar to distinguish these cases!

Reply:
Fixed the issue of post increment having higher precedence. Post increment associativity probably doesn't matter as i++++ doesn't make sense anyway